charts/opentelemetry-operator/values.yaml

```diff
@@ -221,7 +221,10 @@ admissionWebhooks:
   ## TLS Certificate Option 2: Use Helm to automatically generate self-signed certificate.
   ## certManager must be disabled and autoGenerateCert must be enabled.
   ## If true and certManager.enabled is false, Helm will automatically create a self-signd cert and secret for you.
-  autoGenerateCert: true
+  autoGenerateCert:
+    enabled: true
+    # If set to true, new webhook key/certificate is generated on helm upgrade.
+    recreate: false
```

Comment:
If we do this as a breaking change we need to add an entry in UPGRADING.md

Reply:
My mistake, it should be True, fixed.